### DISCUSSION TITLE
Source code of EcuTweaker added.

### DISCUSSION THREAD

**Discussion by Furtif** [cedricp/ddt4all#843] [General]
https://github.com/cedricp/ddt4all/tree/master/EcuTweaker

https://github.com/cedricp/ddt4all#new-wiki-android-port

**Comment by KarelSvo** [comment]
https://github.com/cedricp/ddt4all/archive/refs/heads/master.zip

Anyone who downloads the master branch must take the Ecu Tweaker source with them, which they do not need for ddt4all.
That is illogical.

**Reply by Furtif** [reply]
Yep but uses same ecu.zip

**Reply by KarelSvo** [reply]
Yes, the Ecutweaker, but not its source files, the 'normal' ddt4all user doesn't need them.

**Comment by cedricp** [comment]
I agree, ecutweaker is a separate project

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/843

**Comment by Furtif** [comment]
removed https://github.com/cedricp/ddt4all/commit/547b4e8e018ae4f1849558de64e541726e2a1b41
